.github/workflows/validate_packs.yml

```
@@ -18,8 +18,15 @@ jobs:
       - name: Install dependencies
         run: cd utils && yarn install
 
-      - name: Validate files
-        run: cd utils && yarn validate-quickstarts
+      # - name: Validate files
+      #   run: cd utils && yarn validate-quickstarts
+
+      - name: Validate new files
```

```suggestion
      - name: Validate PR files
```
As we work on this, I'm feeling like we should rename this step as above and rename the related util file to `validate-pr-quickstarts`